File: java/AppOpenExample/app/src/main/java/com/pubscale/sdkone/example/appopenexample/MainActivity.java
package com.pubscale.sdkone.example.appopenexample;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.pubscale.sdkone.core.app_open_ads.general.GGAppOpenAds;
import com.pubscale.sdkone.example.appopenexample.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.switchAppOpen.setOnCheckedChangeListener((buttonView, isChecked) -> {
            GGAppOpenAds.INSTANCE.setShouldShowOnAppMovedToForeground(isChecked);
        });
    }
}

File: kotlin/AppOpenExample/app/src/main/java/com/pubscale/sdkone/example/appopenexample/MainActivity.kt
package com.pubscale.sdkone.example.appopenexample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.pubscale.sdkone.core.app_open_ads.general.GGAppOpenAds
import com.pubscale.sdkone.example.appopenexample.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.switchAppOpen.setOnCheckedChangeListener { _, isChecked ->
            GGAppOpenAds.setShouldShowOnAppMovedToForeground(isChecked)
        }
    }

}